src/content/advanced/vsphere-kubevip/index.md

```diff
@@ -0,0 +1,91 @@
+---
+linkTitle: vSphere Kube-vip
+title: vSphere networking with Kube-vip
+description: Here we describe how to work with Kube-vip in Giant Swarm CAPV clusters.
+weight: 25
+menu:
+  main:
+    parent: advanced
+last_review_date: 2023-10-09
+user_questions:
+  - How to slice my subnet for Kube-vip with CAPV?
+  - What IP should I use for Kube-vip with CAPV?
+  - How to set a range of IPs for load balancer in Kube-vip with CAPV?
+  - Where can I see the IPs in use by Kube-vip?
+aliases:
+  - /guides/kube-vip-for-capv/
+owner:
+  - https://github.com/orgs/giantswarm/teams/team-rocket
+---
+While most infrastructure providers offer a load balancer to use for the Kubernetes API and services of type Load Balancer (AWS, Azure, VMware Cloud Director...), vSphere is a bit different in that it is a hypervisor as opposed to a cloud provider. Advanced networking use cases such as routing, firewalling, load balancing and the likes are handled by VMware NSX. In order to offer multi-control plane nodes to all vSphere customers (even those without NSX), CAPV comes with `Kube-vip`, a layer-2 load balancer.
+
+As a result of working on layer 2, `Kube-vip` uses IP addresses in the same subnet as the nodes themselves and these IP addresses must be selected carefully to avoid collisions. This is particularly plausible if multiple operators collaborate on the platform.
```

Comment:
> This is particularly plausible if multiple operators collaborate on the platform.

Which operators? Humans, software? Let's rather give an example so customers can understand in which cases there's really a risk.

Reply:
done